features/polyExplorer/src/components/clusterStories/baseClusterStory/baseClusterStory.css

```diff
@@ -0,0 +1,48 @@
+.cluster-story .content {
+    width: 100%;
+    color: var(--color-text-dark);
```

Comment:
We already talked about this in the meeting, and it's not within scope, but we should maybe try and avoid appearance values in the variables, using instead "color-text-foreground/background" for instance.